doc/api/dns.md

```diff
@@ -186,14 +186,14 @@ const options = {
   hints: dns.ADDRCONFIG | dns.V4MAPPED,
 };
 dns.lookup('example.com', options, (err, address, family) =>
-  console.log('address: %j family: IPv%s', address, family));
-// address: "2606:2800:220:1:248:1893:25c8:1946" family: IPv6
+  console.log('Address: %j family: IPv%s', address, family));
```

Comment:
Nit, but this change feels weird.  Not all of the `console.log` calls are changed.  Nor even all of the `console.log('Address:...` statements, nor even all of the `console.log('Address..` statements that are followed by a comment, but only the `console.log('Address..` statements that are followed by a comment that is long enough.  At a minimum, any given web page should be self-consistent.

Reply:
I am not that happy with this change either but I don't really know a good solution here. I could just add a ignore statement instead. I could also exclude lines that follow a `console` statement.

Would either work for you or do you have any alternative suggestion?

Reply:
Ignoring markdown files was the only alternative that I could think of at the time, but excluding lines after a `console` statement sounds good to me.

I doubt an ignore statement would work well, given that this source is markdown.

Reply:
It's not possible to match more than a single line but I improved the ignore pattern a bit and all matches look fine to me now.